libs/sdk-ui-charts/src/interfaces/chartConfig.ts

```diff
@@ -311,6 +311,11 @@ export type ChartAlignTypes = "top" | "bottom" | "middle";
 export interface IDataLabelsConfig {
     visible?: IDataLabelsVisible;
     totalsVisible?: IDataLabelsVisible;
+    /**
+     * Indicates whether to render data labels enriched with percentages.
+     * Applies only to some of the charts, funnel chart for example.
```

Comment:
is only funnel chart or also some others? If only Funnel I would say it directly

Reply:
Funnel chart directly specified.